packages/zowe-explorer/src/shared/context.ts

```diff
@@ -150,6 +152,15 @@ export function isMigrated(node: TreeItem): boolean {
     return new RegExp("^(" + globals.DS_MIGRATED_FILE_CONTEXT + ")").test(node.contextValue);
 }
 
+/**
+ * Helper function which identifies if the node has an imperative error
+ * @param node
+ * @return true if there was an error obtaining information about this dataset, false otherwise
+ */
+export function hasImperativeError(node: TreeItem): boolean {
+    return new RegExp("^(" + globals.DS_IMPERATIVE_ERROR_CONTEXT + ")").test(node.contextValue);
+}
```

Comment:
Thanks for renaming the icon 🙂 Should these be renamed to `hasFileError` and `globals.DS_FILE_ERROR_CONTEXT` for consistency? Or do we want to keep this specific to Imperative errors because we might define a new context type for other kinds of errors in the future?